sales/send_quotations/terms_and_conditions.rst

```diff
@@ -0,0 +1,41 @@
+================================
+Add terms & conditions on orders
+=================================
```

Comment:
Too long line,  you could spot it on the build logs.

/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getpaidtovalidate.rst:1: SEVERE: Title overline & underline mismatch.

============================
Get paid to confirm an order
=============================
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getpaidtovalidate.rst:20: ERROR: Unknown target name: "paypal <https://www.odoo.com/documentation/user/11.0/ecommerce/shopper_experience/paypal.html>".
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getpaidtovalidate.rst:20: ERROR: Unknown target name: "authorize.net (pay by credit card) <https://www.odoo.com/documentation/user/11.0/ecommerce/shopper_experience/authorize.html>".
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/getsignaturetovalidate.rst:1: SEVERE: Title overline & underline mismatch.

===================================
Get a signature to confirm an order
====================================
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/terms_and_conditions.rst:1: SEVERE: Title overline & underline mismatch.

================================
Add terms & conditions on orders
=================================
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/terms_and_conditions.rst:20: ERROR: Error in "image" directive:
invalid option block.

.. image:: media/terms_and_conditions01.png
  :align:
  center
/home/odoo/odoo_dev/documentation-user/sales/send_quotations/terms_and_conditions.rst:31: WARNING: Explicit markup ends without a blank line; unexpected unindent.

Reply:
This kind of errors are critical. Could you fix ? Thanks.